### PR TITLE
Add SVGInjector@1.1.3 with npm autoupdate

### PR DESCRIPTION
**Pull request by LeaYeh**
**git repo url:** https://github.com/iconic/SVGInjector
**Watch 26
 Star 419
 Fork 53**
@Amomo   could you help me check out the PR for #6352  ? thank you~
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/SVGInjector/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "filename": "svg-injector.min.js",
+  "name": "SVGInjector",
```
Comment (Amomo) [inline ajax/libs/SVGInjector/package.json]: According to its package.json & bower.json, I think we should use svg-injector.

File: ajax/libs/SVGInjector/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "SVG",
+    "Scalable Vector Graphics",
+    "SVG injector",
+    "images",
+    "img",
+    "html",
+    "DOM"
+  ],
+  "license": "MIT",
+  "autoupdate": {
```
Comment (Amomo) [inline ajax/libs/SVGInjector/package.json]: I think use npm auto-update would be nicer.

**Comment by LeaYeh** [comment]
why not use this as proj name?

**Comment by Amomo** [comment]
@LeaYeh 

In my opinion, use the name in package.json as folder name is more safety.
Because the name of repo may change, but the name in package.json(especially it has released on npm)  usually won't change.

And, we also provide the repo name on our web page.

**Comment by PeterDaveHello** [comment]
Both fine, but I won't touch it if there is already a name key-value info in package.json, unless it's a strange case.

**Comment by LeaYeh** [comment]
so i just follow the origin `name` ?

**Comment by PeterDaveHello** [comment]
Yep.

**Comment by LeaYeh** [comment]
@Amomo could you check it again for me?

**Comment by Amomo** [comment]
@LeaYeh 
Sorry for late, I think this PR is fine, could you rebased it to latest branch?
Thank you! :-)

**Comment by LeaYeh** [comment]
@Amomo i rebased it ! could you check it for me?

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look? Thank you! :-D

**Comment by PeterDaveHello** [comment]
Let's use npm auto-update.

**Comment by LeaYeh** [comment]
@PeterDaveHello  okay, i use npm config instead of git config and rebased it.

**Comment by PeterDaveHello** [comment]
So why don't you change the commit message?
> 	Add SVGInjector@1.1.3 with git autoupdate …

**Comment by LeaYeh** [comment]
@PeterDaveHello i change commit msg and pr title to npm config.

**Pull request merged**